src/fizzBuzz.js

```diff
@@ -0,0 +1,19 @@
+export default function fizzBuzz(N) {
+  const buzzArray = []
+  for (let i = 1;  i <= N; i++) {
+    let pushVal = ''
+
+    if(i % 3 === 0) {
+      pushVal = pushVal.concat('Fizz')
+    }
+    if(i % 5 === 0) {
+      pushVal = pushVal.concat('Buzz')
+    }
+    if(pushVal === '') {
+      pushVal = i
+    }
+    buzzArray.push(pushVal)
+  }
+
+  return buzzArray
+}
```

Comment:
This is the way I had implemented this when I had the project, but the way you did it works. 
I'm just showing you for comparison.:

``` const fizzBuzz = () => {
  const array = []

  for (let i = 0; i <= 100; i++) {
    if (i === 0) {
      array.push(++i)
    } else if (i % 3 === 0 & i % 5 === 0) {
      array.push('FizzBuzz')
    } else if (i % 3 === 0) {
      array.push('Fizz')
    } else if (i % 5 === 0) {
      array.push('Buzz')
    } else {
      array.push(i)
    }
  }

  return array
}
```